### PR TITLE
E-1024 feat: improve login button design

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: src/css/fonts.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: 'JetBrains Mono';
+  src: local('JetBrains Mono Regular'),
```
Comment (lamartire) [inline src/css/fonts.scss:3]: Add `local('JetBrainsMono-Regular')` too

File: src/css/typography.scss [hunk 2]
```diff
@@ -10,6 +10,11 @@ h6 {
   font-family: $heading-font-family;
 }
 
+body,
+input {
```
Comment (lamartire) [inline src/css/typography.scss:14]: Add `textarea` and `select`

File: src/components/screen/Oauth/Oauth.vue [hunk 3]
```diff
@@ -89,11 +116,18 @@ export default {
...
     PopupModeSwitcher,
     ContentLoader,
     VHeaderControls,
     VCard,
   },
 };
 </script>
+
+<style lang="scss">
+.oauth-section {
```
Comment (lamartire) [inline src/components/screen/Oauth/Oauth.vue:130]: `:not(:last-child)`

**Comment by ifiri** [comment]
@dudiq is this okay? I did check Youtrack, is these button should not be aligned horizontally?
image

**Comment by dudiq** [comment]
@ifiri had some fixes, please make thread not here, but in files section

**Comment by ifiri** [comment]
@kazurus approved

**Pull request merged**